project-koku/koku
# use schema on raw delete

Fixes [COST-####](https://issues.redhat.com/browse/COST-####)

Changes proposed in this PR:
* specifically set the schema for the delete SQL in the raw delete function
*

Testing Instructions:
1. Spin up koku, such as: `make docker-up-min-trino`
2. Load the test customer data such as: `make create-test-customer` followed by `make load-test-customer-data`
3. See some deletes in the logs with the message from the raw deletes like this:
```
[2022-03-01 15:47:33,053] INFO 27795372-9cb3-42ea-8346-18af9ae3afe3 Deleting records from reporting_gcpcostentrylineitem_daily_summary from 2022-03-01 to 2022-03-01
```
4. processing goes smoothly and there are no issues and you have data

---
Additional Context:
